Inputs/config.js

```diff
@@ -20,104 +20,134 @@ const accountSetting = {
-  "AAVE/USDT",
-  "QNT/USDT",
-  "AXS/USDT",
-  "FIL/USDT",
-  "ETH/USDC",
-  "BNB/USDC",
-  "BTC/USDC",
-  "XRP/USDC",
-  "ADA/USDC",
-  "BAT/USDC",
-  "LINK/USDC",
-  "SOL/USDC",
-  "TRX/USDC"
+let symbols = [
+  // 'USDC/USDT',
+  // 'ALGO/USDT',
+  // "ETH/USDT",
+  // "BNB/USDT",
+  "BTC/BUSD"
+  // "XRP/USDT",
+  // "ADA/USDT",
+  // "TRX/USDT",
+  // "BAT/USDT",
+  // "LINK/USDT",
+  // "MANA/USDT",
+  // "SOL/USDT",
+  // "MATIC/USDT",
+  // "THETA/USDT",
+  // "HNT/USDT",
+  // "DOT/USDT",
+  // "YFI/USDT",
+  // "LUNA/USDT",
+  // "UNI/USDT",
+  // "AVAX/USDT",
+  // "AAVE/USDT",
+  // "QNT/USDT",
+  // "AXS/USDT",
+  // "FIL/USDT",
+  // "ETH/USDC",
+  // "BNB/USDC",
+  // "BTC/USDC",
+  // "XRP/USDC",
+  // "ADA/USDC",
+  // "BAT/USDC",
+  // "LINK/USDC",
+  // "SOL/USDC",
+  // "TRX/USDC"
 ]
 
-const symbolsForMarkPrice = [
-  'USDC/USDT',
-  'ALGO/USDT',
+let symbolsForMarkPrice = [
   "ETH/USDT",
-  "BNB/USDT",
-  "BTC/USDT",
-  "XRP/USDT",
-  "ADA/USDT",
-  "TRX/USDT",
-  "BAT/USDT",
-  "LINK/USDT",
-  "MANA/USDT",
-  "SOL/USDT",
-  "MATIC/USDT",
-  "THETA/USDT",
-  "HNT/USDT",
-  "DOT/USDT",
-  "YFI/USDT",
-  "LUNA/USDT",
-  "UNI/USDT",
-  "AVAX/USDT",
-  "AAVE/USDT",
-  "QNT/USDT",
-  "AXS/USDT",
-  "FIL/USDT"
+  // "BNB/USDT",
+  "BTC/USDT"
+  // 'USDC/USDT',
+  // 'ALGO/USDT',
+  // "ETH/USDT",
+  // "BNB/USDT",
+  // "BTC/USDT",
+  // "XRP/USDT",
+  // "ADA/USDT",
+  // "TRX/USDT",
+  // "BAT/USDT",
+  // "LINK/USDT",
+  // "MANA/USDT",
+  // "SOL/USDT",
+  // "MATIC/USDT",
+  // "THETA/USDT",
+  // "HNT/USDT",
+  // "DOT/USDT",
+  // "YFI/USDT",
+  // "LUNA/USDT",
+  // "UNI/USDT",
+  // "AVAX/USDT",
+  // "AAVE/USDT",
+  // "QNT/USDT",
+  // "AXS/USDT",
+  // "FIL/USDT"
 ]
 
-const symbolsForSettlementPrice = [
-  'USDC/USDT',
-  'ALGO/USDT',
+let symbolsForSettlementPrice = [
   "ETH/USDT",
-  "BNB/USDT",
-  "BTC/USDT",
-  "XRP/USDT",
-  "ADA/USDT",
-  "TRX/USDT",
-  "BAT/USDT",
-  "LINK/USDT",
-  "MANA/USDT",
-  "SOL/USDT",
-  "MATIC/USDT",
-  "THETA/USDT",
-  "HNT/USDT",
-  "DOT/USDT",
-  "YFI/USDT",
-  "LUNA/USDT",
-  "UNI/USDT",
-  "AVAX/USDT",
-  "AAVE/USDT",
-  "QNT/USDT",
-  "AXS/USDT",
-  "FIL/USDT"
+  // "BNB/USDT",
+  "BTC/USDT"
+  // 'USDC/USDT',
+  // 'ALGO/USDT',
+  // "ETH/USDT",
+  // "BNB/USDT",
+  // "BTC/USDT",
+  // "XRP/USDT",
+  // "ADA/USDT",
+  // "TRX/USDT",
+  // "BAT/USDT",
+  // "LINK/USDT",
+  // "MANA/USDT",
+  // "SOL/USDT",
+  // "MATIC/USDT",
+  // "THETA/USDT",
+  // "HNT/USDT",
+  // "DOT/USDT",
+  // "YFI/USDT",
+  // "LUNA/USDT",
+  // "UNI/USDT",
+  // "AVAX/USDT",
+  // "AAVE/USDT",
+  // "QNT/USDT",
+  // "AXS/USDT",
+  // "FIL/USDT"
 ]
 
 const symbolLastUpdate = {}
 
+const fetchActiveSymbol = async(client)=>{
+  const query = `select distinct symbol from dbt_traderdao.activeasset`;
+  const res = await client.query(query);
+  const newAsset = res.rows;
+  // await client.end()
+  // console.log('newAsset', newAsset)
+  const tradingQuotation = newAsset.map((symbol)=>{
```

Comment:
We usually name array with ```s```,for example, tradingQuotations, so we can know it's array from its name.